client/app.jsx

```diff
@@ -22,19 +29,35 @@ export default class App extends React.Component {
       .catch(err => console.error('GET error', err.message));
   }
 
-  render() {
+  renderPage() {
+    const prodId = Number(this.state.route.id);
+    const { route } = this.state;
     const products = this.state.products;
+
+    if (route.path === '') {
+      return (
+      <div className='row'>
+      {
+    products.reverse().map((product, index) => {
+      return <NewItems key={index} product={product} />;
+    })
```

Comment:
Why reversed? Your server should probably send you the data in the order you need it.